docs/configuration/CONFIGURATION.md

```diff
@@ -45,36 +45,176 @@ This overview is still a work in progress. You can refer to the [example.config.
 
 :::
 
-### URL
+### Application URL
 
-UniCore's base URL. This value is communicated to clients and identity wallets.
+UniCore's application URL. This value represents the self-aware URL of the application. It is used for internal
+communication and should not be exposed to clients or identity wallets.
```

Comment:
I think we should be a bit more precise here. Why _should_ it not be exposed? For security reasons? We can for example say: **"This is the URL that you can use for routing behind a gateway or reverse proxy"**. I also think we should add a reference to the "Public URL" in this description already to avoid confusion: **"This might differ from from the URL that UniCore should use for external communication with wallets and other services"**.